sdk/eventgrid/tests.yml

```diff
@@ -5,6 +5,7 @@ stages:
     parameters:
       ServiceDirectory: eventgrid
       BuildTargetingString: azure-eventgrid*
+      Location: centraluseuap
```

Comment:
set location to centraluseuap -- currently only canary supports the mgmt api version for deploying resources -- will update when deployed fully